site/server.js

```diff
@@ -43,11 +45,37 @@ try {
 } catch (e) {
   throw "No config.js file found. Please follow the format in config.js.example";
 }
- 
 if (!config.api_key) {
   throw "You need an API key in config.js in order to run this program. Please add to config.js";
 }
 
+// Get social config
+var regex_tweet = new RegExp('^.*' + 'TWEET="' + '.*$', 'm');
+var regex_event_name = new RegExp('^.*' + 'EVENT_NAME="' + '.*$', 'm');
+
+fs.exists(CONFIG_PATH, function(fileok) {
+  if (fileok) {
+    fs.readFile(CONFIG_PATH, "utf8", function(err, data) {
+      var t = regex_tweet.exec(data);
+      var e = regex_event_name.exec(data);
+
+      if (t) {
+        var line = t[0];
+        socialConfig[0] = line.split('TWEET=')[1];
+      } else {
+        socialConfig[0] = "Thanks for visiting the @GCPEmotobooth! See all photos and data from this session → ";
```

Comment:
I would create the `socialConfig` array on line 35 with the default values already in there and then only have to care about the `if` cases here, but that's a personal style thing :)